### PR TITLE
Fixes around backticks

### PR DESCRIPTION
**Pull request by svick**
I have searched the docs for uninterrupted sequences of letter-backtick-letter and fixed those that I though were wrong. That means:

* Adding a space before and after code.
* Moving backtick that's incorrectly in the middle of a word.
* Other fixes.

### CODE REVIEW

File: docs/framework/tools/al-exe-assembly-linker.md
Comment (GuardRex) [inline docs/framework/tools/al-exe-assembly-linker.md]: @svick I see a few little things on this topic not directly relevant to your PR. I'll get 'um ... I'm looking for some small fill-in thingies to do before I go. I don't want to hold this set of changes up. I'll work it on:  https://github.com/dotnet/docs/issues/2739

File: docs/framework/tools/ildasm-exe-il-disassembler.md
Comment (GuardRex) [inline docs/framework/tools/ildasm-exe-il-disassembler.md]: Same here ... I plan to have a follow-up look at this later in the week on: https://github.com/dotnet/docs/issues/2739

**Review by rpetrusha — APPROVED** [review]
Thanks for making all of these changes, @svick! We'll merge after the update to live PR builds and is merged.

**Review by guardrex — APPROVED** [review]
@rpetrusha Ooops ... I thought I'd save u some time by reviewing. I'm opening a pair of side issues for some things to fix not directly related here, but I reached the same result u did on the review.

File: docs/framework/data/adonet/ef/string-functions.md
Comment (svick) [inline docs/framework/data/adonet/ef/string-functions.md]: @GuardRex Done.

**Comment by GuardRex** [comment]
Thanks @svick.

**Pull request merged**
